### DISCUSSION TITLE
[Unstable] Gravity shell does not take effect indoors

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14639] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As the title said.

https://github.com/user-attachments/assets/53366183-4e8b-48ab-96ac-f9dbef8ee637

### Reproduction steps

1. Fire a gravity shell indoor

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

Other

### -

v1.6.6.0

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
And oddly enough gravity spheres don't work outside of hulls

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in the next unstable build!
